new-dti-website/src/app/apply/page.tsx

```diff
@@ -6,19 +6,21 @@ import config from '../../../config.json';
 import RoleDescriptions from '../../../components/apply/RoleDescription';
 import RedBlob from '../../../components/blob';
 import ApplyFAQ from '../../../components/apply/ApplyFAQ';
+import Banner from '../../../components/apply/Banner';
 import useThemeContext from '../../hooks/useThemeContext';
 import { isAppOpen } from '../../utils/dateUtils';
 
 const ApplyHero = () => (
-  <div className="text-[#FEFEFE] min-h-[calc(100vh-136px)] flex items-center">
+  <div className="text-[#FEFEFE] min-h-[calc(100vh-136px)] flex items-center relative">
```

Comment:
Could you provide a screen recording of how this looks on mobile too please?

Reply:
<img width="421" alt="Screenshot 2024-11-21 at 8 53 32 PM" src="https://github.com/user-attachments/assets/c23dd483-9268-4183-bcbb-0cb7c6003720">

This is what it looks like right now for mobile!

Reply:
ok nice!